package.json

```diff
@@ -7,9 +7,9 @@
     "lint": "tslint --project . --fix",
     "build": "rimraf dist && tsc --project tsconfig.dist.json",
     "test": "jest --coverage --runInBand --verbose",
-    "test:ci": "jest --coverage --coverageReporters=text-lcov | coveralls",
+    "test:ci": "jest --coverage --coverageReporters=text-lcov",
```

Comment:
Perhaps we should consolidate these so we don't need `test:ci` as a separate target?